### PR TITLE
ceph/admin: s/master/main

### PR DESCRIPTION
**Pull request by zdover23**
This PR changes the name "master" to "main" so
that builds (and, I assume, a great many other
things) will not fail.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by jdurgin — APPROVED** [review]

**Comment by tchaikov** [comment]
jenkins test make check

File: admin/doc-read-the-docs.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 plantweb
-git+https://github.com/readthedocs/readthedocs-sphinx-search@master
+git+https://github.com/readthedocs/readthedocs-sphinx-search@main
```
Comment (tchaikov) [inline admin/doc-read-the-docs.txt:2]:
```suggestion
git+https://github.com/readthedocs/readthedocs-sphinx-search@0.1.1
```

**Review by tchaikov — CHANGES_REQUESTED** [review]

**Comment by tchaikov** [comment]
@zdover23 ping? it's still an improvement without incorporating my suggestion. if you are fine with it but don't have the bandwidth working on it. i can help on the squash things.

**Comment by zdover23** [comment]
> @zdover23 ping? it's still an improvement without incorporating my suggestion. if you are fine with it but don't have the bandwidth working on it. i can help on the squash things.

@tchaikov -- sorry about the radio silence on my part. I've been at capacity for the last four days, and only now got back to this. It might look me-too-ish of me to say this, but I agree with Josh about making changes like this as incrementally as possible. (Sorry inb4 Josh, if I'm putting words into your mouth.)

**Pull request merged**